rmd/customConcept.Rmd

```diff
@@ -0,0 +1,93 @@
+---
+title: "**Custom Concepts**"
+output: html_document
+---
+
+*By Maxim Moinat and the Themis Working Group*
+
+The Themis Working Group convened on October 6th and December 7th 2023 to discuss the creation this convention for creating custom concepts.
+
+## Introduction
+While the OMOP vocabularies are very comprehensive, it is not always possible to use concepts existing in the OMOP vocabularieas. For example, when using a vocabulary that is only used in your institution or having custom defined variables. In these cases, custom concepts can be used. Custom concepts are concepts that are not part of the OMOP vocabularies, and are only used in your institution.
```

Comment:
So we are defining custom concepts as never being part of OMOP vocabularies? This assumes that the missing concept is not going to be re-used and only has value to the local institution. The Jackalope concept creates custom concepts that are precoordinated standard concepts. These clearly could be re-used. Do we want to distinguish up front that our definition of "custom" excludes these other use cases for creating concepts?

Reply:
To me, custom concepts and standard concepts are mutually exclusive. Adding concepts (as standard OR non-standard) to the OHDSI vocabularies is a separate process.

There are several subtleties of course, which we might want to address or not:
- A method to go from custom concepts to a contribution to the OHDSI vocabularies. Only at this point a concept_id <2B would be assigned and possibly made standard. Always in collaboration with the Vocabulary WG.
- Once your source vocabulary has been added to the OHDSI vocabularies, then of course your mapping strategy will change and you might not have a need for custom concepts anymore.
- Custom concepts will always be more flexible than adding your concepts to the OHDSI vocabularies, as you don't have to go through the vocabularies release process.